pkg/apis/servicecatalog/types.go

```diff
@@ -211,6 +210,11 @@ type InstanceStatus struct {
 	// AsyncOpInProgress is set to true if there is an ongoing async operation
 	// against this Service Instance in progress.
 	AsyncOpInProgress bool
+
+	// LastOperation is the string that the broker may have returned when
+	// an async operation started, it should be sent back to the broker
+	// on poll requests as a query param.
+	LastOperation string
```

Comment:
@luksa We are going to be renaming the fields to remove the `OSB` prefix where they still exist.  We prefixed them initially to call out where things were specific to the OSB API, because we weren't certain whether we were going to support another API (like APPS+ODIN).  At this point, we're just going to support OSB, so we can make the fields have names that are a little better.  Remember, this API is alpha, so you should expect renames / changes in the API.